Cakefile

```diff
@@ -3,23 +3,28 @@ fs     = require 'fs'
 async  = require './www-src/vendor/scripts/async'
 
 plugins = {
+    "cordova-plugin-whitelist": "cordova-plugin-whitelist@1.0.0"
+
+    "cordova-plugin-device": "cordova-plugin-device@1.0.1"
+    "cordova-plugin-globalization": "cordova-plugin-globalization@1.0.1"
+    "cordova-plugin-inappbrowser": "cordova-plugin-inappbrowser@1.0.1"
+    "cordova-plugin-file": "cordova-plugin-file@3.0.0"
+    "cordova-plugin-file-transfer": "cordova-plugin-file-transfer@1.3.0"
+    "cordova-plugin-battery-status": "cordova-plugin-battery-status@1.1.0"
+    "cordova-plugin-network-information": "cordova-plugin-network-information@1.0.1"
+
     "com.fgomiero.cordova.externafileutil": "https://github.com/aenario/cordova-external-file-open"
     "com.brodysoft.sqlitePlugin": "https://github.com/brodysoft/Cordova-SQLitePlugin#r1.0.4"
-    "org.apache.cordova.file": "https://git-wip-us.apache.org/repos/asf/cordova-plugin-file.git#r1.3.3"
-    "org.apache.cordova.file-transfer": "https://git-wip-us.apache.org/repos/asf/cordova-plugin-file-transfer.git#r0.5.0"
+    "de.appplant.cordova.plugin.local-notification": "de.appplant.cordova.plugin.local-notification@0.8.2"
+
     "io.cozy.cordova-images-browser": "https://github.com/aenario/cordova-images-browser"
-    "io.cozy.jsbackgroundservice": "https://github.com/jacquarg/cordova-jsbackgroundservice#v1.0.1"
+    "io.cozy.jsbackgroundservice": "https://github.com/jacquarg/cordova-jsbackgroundservice"
```

Comment:
No. I am still debuging a new version of this plugin. I expect to move to v2.0.0 when we will be ready to release.